### PR TITLE
Lighthouse accessibility to 100

### PR DESCRIPTION
**Pull request by rdsilver**
I've spruced up our lighthouse score to 100.

**Before**:
Screen Shot 2021-12-14 at 9 24 55 PM

**After**:
Screen Shot 2021-12-14 at 9 24 04 PM

### CODE REVIEW

**Review by jongregis — APPROVED** [review]

**Comment by clifton** [comment]
thank you!

File: components/subscribe-form.tsx [hunk 1]
```diff
@@ -110,7 +110,9 @@ const SubscribeForm: FC = () => {
             Sign up for dao updates
           </Button>
           <FormControl hidden={status !== 'unsubmitted'}>
-            <FormHelperText>We will never share your email.</FormHelperText>
+            <FormHelperText color="gray.600">
```
Comment (clifton) [inline components/subscribe-form.tsx:113]: might look off in dark mode but really happy someone else is familiar with chakra!

**Comment by clifton** [comment]
@jongregis need to wait for the green check mark b4 merge

File: components/subscribe-form.tsx (hunk 1 above)
Comment (rdsilver) [reply components/subscribe-form.tsx:113]: 😅  I just looked up the docs. At my main job we use styled components :).

Comment (clifton) [reply components/subscribe-form.tsx:113]: i believe this is styled-components under the hood (tho i think there  might be another layer in there).

**Pull request merged**